recipes/biapy/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set name = "biapy" %}
+{% set version = "3.6.2" %}
+{% set python_min = "3.10" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/BiaPyX/BiaPy/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 7257443e4cd0d46f82dd9b266a8893e9809a02446572e06184e82457f910935c
+
+build:
+  noarch: python
+  number: 0
+  script: pip install . --no-deps --no-build-isolation
+  entry_points:
+    - biapy = biapy:main
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - pip
+    - setuptools
+  run:
+    - python >={{ python_min }}
+    - pytorch >=2.4
+    - timm ==1.0.14
+    - pytorch-msssim
+    - torchmetrics >=1.4,<1.5
+    - matplotlib-base >=3.7.1
+    - scikit-learn >=1.4.0
+    - pydot >=1.4.2
+    - yacs >=0.1.8
+    - tqdm >=4.66.1
+    - scikit-image >=0.21.0
+    - edt >=2.3.2
+    - numpy >1.26,<2 # BiaPy migrated to numpy 2 already but pinning this just to ensure the conda package can build as they are still doing the migration
+    - fill-voids >=2.0.6
+    - opencv >=4.8.0
+    - pandas >=1.5.3
+    - torchinfo >=1.8.0
+    - tensorboardX >=2.6.2.2
+    - h5py >=3.9.0
+    - zarr >=2.16.1,<3.0
+    - bioimageio.core ==0.8.0
+    - imagecodecs >=2024.1.1
+    - pooch >=1.8.1
+    - xarray ==2025.1.*
+
+test:
+  imports:
+    - biapy
+  commands:
```

Comment:
You probably want a `pip check`:
https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml#L66-L67